osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -196,5 +197,8 @@ public enum GlobalAction
 
         [Description("Random Skin")]
         RandomSkin,
+
+        [Description("Pause Replay")]
+        PauseReplay,
```

Comment:
Maybe better wording as it can resume playback. Also descriptions were once sentence cased, the recent ones somehow stayed title case.

```suggestion
        [Description("Toggle replay playback")]
        ToggleReplayPlayback,
```

Reply:
I think I'd propose `Pause / Resume replay` or similar rather than "toggle", the same way we have `Play / Pause` on `GlobalAction.MusicPlay`. I feel the latter is a little overly verbose but */shrug*.